### ISSUE TITLE
Accuracy of pretrained model

### ISSUE THREAD

**Issue by whuoyj** [kenshohara/3D-ResNets-PyTorch#198]
Hello,
I used your pretrained model _resnet-50-kinetics.pth_ and _r3d50_M200ep.pth_ to evaluate the validation set of kinetics dataset and moments in time dataset.
But I got accuracy both **less than 1%**, I was wonder the problem is the output class label order of the pretrained model is different from mine, could you please give me your **kinetics.json** and **mit.json** file you have used when you training these pretrained model to let me check the order of class label ?

Examples for evaluate moments in time dataset:

_python main.py --root_path ~/data --video_path mit_videos/jpg --annotation_path mit.json --result_path results --dataset mit --resume_path results/r3d50_M_200ep.pth --model_depth 50 --n_classes 339 --n_threads 4 --no_train --no_val --inference --output_topk 5 --inference_batch_size 1_

My class label order in **mit.json** is: {"labels": ["clapping", "praying", "dropping", "burying", "covering", "flooding"...
But with the Inference process, I got the results in val.json: {"results": {"yt--16CPtUiaX4_270": [{"label": "starting", "score": 0.3101961314678192}, {"label": "eating", "score": 0.30093416571617126}, {"label": "rinsing", "score": 0.050913989543914795}, {"label": "frowning", "score": 0.035624679177999496}, {"label": "welding", "score": 0.03383404016494751}], "yt-4IPc6L27rE8_35": [{"label": "starting", "score": 0.33459505438804626}, {"label": "inflating", "score": 0.1385657638311386}, {"label": "baking", "score": 0.06203879415988922}, {"label": "pitching", "score": 0.053437795490026474}, {"label": "calling", "score": 0.027744652703404427}]...

**Comment by cinjon** [comment]
Also having this issue.

**Comment by villawang** [comment]
Have you solved your problem? I also got a very low testing performance.
